manager/integration/tests/test_basic.py

```diff
@@ -1326,34 +1304,6 @@ def restore_inc_test(client, core_api, volume_name, pod):  # NOQA
     assert not k_status.lastPVCRefAt
     assert not k_status.lastPodRefAt
 
-    delete_and_wait_pod(core_api, sb_volume2_pod_name)
-    delete_and_wait_pvc(core_api, sb_volume2_name)
-    delete_and_wait_pv(core_api, sb_volume2_name)
-
-    # cleanup
-    std_volume.detach()
-    sb_volume0.detach()
-    sb_volume1.detach()
-    std_volume = common.wait_for_volume_detached(client, volume_name)
-    sb_volume0 = common.wait_for_volume_detached(client, sb_volume0_name)
-    sb_volume1 = common.wait_for_volume_detached(client, sb_volume1_name)
-    sb_volume2 = common.wait_for_volume_detached(client, sb_volume2_name)
-
-    backupstore_cleanup(client)
```

Comment:
It seems that `cleanup_client()` doesn't contain this part. Maybe you need to add it for that function.

Reply:
This cleans up by [set_random_backupstore](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/backupstore.py#L51)...?

Reply:
Oh, sorry for not checking that function... Yeah it's better to update the description